health-sdk/example-app/src/main/java/net/gini/android/health/sdk/exampleapp/review/ReviewActivity.kt

```diff
@@ -1,34 +1,42 @@
 package net.gini.android.health.sdk.exampleapp.review
 
 import android.content.Context
+import android.content.DialogInterface
 import android.content.Intent
 import android.net.Uri
 import android.os.Bundle
 import android.util.Log
+import android.view.View
+import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import androidx.core.view.WindowCompat
 import androidx.core.view.isGone
-import androidx.fragment.app.Fragment
-import androidx.fragment.app.FragmentFactory
 import androidx.fragment.app.commit
+import androidx.lifecycle.Lifecycle
 import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
 import androidx.lifecycle.viewModelScope
 import dev.chrisbanes.insetter.applyInsetter
 import kotlinx.coroutines.cancel
 import kotlinx.coroutines.launch
+import net.gini.android.core.api.models.Document
 import net.gini.android.health.sdk.GiniHealth
+import net.gini.android.health.sdk.bankselection.BankSelectionBottomSheet
 import net.gini.android.health.sdk.exampleapp.R
 import net.gini.android.health.sdk.exampleapp.databinding.ActivityReviewBinding
+import net.gini.android.health.sdk.paymentcomponent.PaymentComponent
+import net.gini.android.health.sdk.paymentcomponent.PaymentProviderAppsState
 import net.gini.android.health.sdk.review.ReviewConfiguration
 import net.gini.android.health.sdk.review.ReviewFragment
 import net.gini.android.health.sdk.review.ReviewFragmentListener
+import net.gini.android.health.sdk.review.model.ResultWrapper
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class ReviewActivity : AppCompatActivity() {
 
     private val viewModel: ReviewViewModel by viewModel()
 
-    private val reviewFragmentListener = object: ReviewFragmentListener {
+    private val reviewFragmentListener = object : ReviewFragmentListener {
         override fun onCloseReview() {
             Log.i("review_events", "on close clicked")
```

Comment:
Do we use Log in the example and Logger(SL4J) in the SDK?

Reply:
Yeees... it's not ideal, isn't it? I will use SLF4J in the example app, too. I will do that in a dedicated commit in the branch for [IPC-160](https://ginis.atlassian.net/browse/IPC-160).